extension/src/popup/views/AccountCreator/index.tsx

```diff
@@ -64,95 +60,97 @@ export const AccountCreator = () => {
-                      <Input
-                        autoComplete="off"
-                        id="new-password-input"
-                        placeholder="New password"
-                        type="password"
-                        error={
-                          authError || (errors.password && touched.password)
-                            ? errors.password
-                            : ""
-                        }
-                        {...field}
-                      />
-                    )}
-                  </Field>
-                </FormRow>
-                <FormRow>
-                  <Field name="confirmPassword">
-                    {({ field }: FieldProps) => (
-                      <Input
-                        autoComplete="off"
-                        id="confirm-password-input"
-                        placeholder="Confirm password"
-                        type="password"
-                        error={
-                          authError ||
-                          (errors.confirmPassword && touched.confirmPassword)
-                            ? errors.confirmPassword
-                            : null
-                        }
-                        {...field}
-                      />
-                    )}
-                  </Field>
-                </FormRow>
-                <PasswordRequirements />
-                <FormRow>
-                  <Field name="termsOfUse">
-                    {({ field }: FieldProps) => (
-                      <Checkbox
-                        autoComplete="off"
-                        id="termsOfUse-input"
-                        label={
-                          <span>
-                            I have read and agree to{" "}
-                            <TextLink
-                              variant={TextLink.variant.secondary}
-                              href="https://stellar.org/terms-of-service"
-                            >
-                              Terms of Use
-                            </TextLink>
-                          </span>
-                        }
-                        {...field}
-                      />
-                    )}
-                  </Field>
-                </FormRow>
-                {/* TODO - add error to Checkbox in SDS */}
-                {errors.termsOfUse && touched.termsOfUse
-                  ? errors.termsOfUse
-                  : null}
-                <div className="AccountCreator--button-row">
-                  <Button
-                    fullWidth
-                    isLoading={isSubmitting}
-                    disabled={!(dirty && isValid)}
-                  >
-                    CONFIRM
-                  </Button>
-                </div>
-              </section>
-            </Form>
-          )}
-        </Formik>
-      </section>
+      <Onboarding goBack={() => navigateTo(ROUTES.welcome)}>
+        <section className="AccountCreator--screen">
```

Comment:
NIT, `--screen` should be `__screen` (element rather than modifier). Same for other elements below.

Reply:
good catch https://github.com/stellar/freighter/pull/361/commits/1a256ecdf0c9f64d638b809fa8e7c191b2307303